Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragment.kt

```diff
@@ -90,6 +90,9 @@ class HomeFragment : Fragment(R.layout.fragment_home), AutoInject {
                 HomeFragmentEvents.ShowDeleteTestDialog -> {
                     showRemoveTestDialog()
                 }
+                HomeFragmentEvents.GoToContactDiary -> {
+                    (requireActivity() as MainActivity).navigateToContactDiaryActivity()
```

Comment:
Why does this go through the activity? Are there are navigation paths besides HomeFragment?

Reply:
https://github.com/corona-warn-app/cwa-app-android/pull/1952